packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -289,6 +289,15 @@ class DynamicSidecarServiceLabels(BaseModel):
         ),
     )
 
+    user_preferences_path: Json[Path] | None = Field(
```

Comment:
Q: Is it necessary to use `Json` here if you are just expecting a `Path`? This should work just with `Path | None` since path is a `string`, right?.  If the path has quotes `"` you can always use a pre-procress validator and `v.strip('"')` them

We also discussed this several times: *we should avoid `Json`*. As a reminder of why, please  see some examples 
https://github.com/ITISFoundation/osparc-simcore/blob/f2f6a81fa9c4021d44c4d77a2fd196a956e6e54b/packages/models-library/tests/test__pydantic_models.py#L61-L64